modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1748,6 +1748,10 @@
 			<category name="Upgrade" />
 			<description name="Replace parameter in updateCommerceAddress method by other parameters list" />
 		</check>
+		<check name="UpgradeJSONLiferayThemeCheck">
+			<category name="Upgrade" />
+			<description name="Replace liferay theme version in package.json" />
```

Comment:
Hi @michaelmdelcavalcanti , maybe this should be more restrictive, what do you think? Is any package.json? Or only the one inside of the theme? If it will be the one inside of a theme we can change this here and inside the check.